### ISSUE TITLE
Some more mods and 4.6

### ISSUE THREAD

**Issue by BobVul** [ctrlcctrlv/wxsand#6]
I found a copy of wxsand in an old backup with some mods I downloaded back then. Not sure if any are customised. Looks like you already have some of them.

[Mods.zip](https://github.com/ctrlcctrlv/wxsand/files/3587358/Mods.zip)

Some of the mods are commented for version 4.5 and 4.6... not sure if they'll work on 4.4.

**Comment by ctrlcctrlv** [comment]
Hi!

Thanks for your contribution. 

I don't think that I can use the 4.6 binary...this repository is meant to host the _source code_, so I'd gladly take the 4.6 source code, but I don't think anyone has that. Regardless, thanks for uploading it, maybe someone is looking for it.

Regarding the mods, thanks! I'll figure out if any of them are not already in the repo.

**Comment by BobVul** [comment]
Yup, I was expecting that. I wish I had the source, but even if it was available I certainly wouldn't have been interested in it back then :)

That said there does seem to be 4.5 here (by the original dev?) https://github.com/opiette/wxsand. Which matches the version on their site. Unfortunately the license is unclear.

I wonder where 4.6 came from? Possibly third-party? It does identify an "Andrew Perry" in About, rather than just Owen Piette. Update search goes to `fallingsandgame.com`, unfortunately, so that's a dead end.

e: they might've gone by `purple` on those forums.

**Comment by ctrlcctrlv** [comment]
Interesting! I've never seen that link before. I originally checked this in on Oct 3, 2015; but that one dates to May 2015. Perhaps if I'd known about it I'd have never created this repo? I hope @opiette doesn't mind; I just wanted to preserve this and share my patch for building it on modern Linux systems.

**Comment by ctrlcctrlv** [comment]
Now that you mention `purple`, I think you're right! I remember `purple` creating his own version, I don't think Owen made 4.6.

**Comment by BobVul** [comment]
Given the unclear licensing around 4.5/4.6, I'll take that link off the original post... the mods should be fine, I hope.

I did just find a copy of 4.6.3 in a newer backup. Still no source though :(

**Comment by ctrlcctrlv** [comment]
Right, astute observation. I know I have the right to redistribute 4.4 because it has a clear license. The 4.5 source you linked above doesn't seem to. I'm not sure what the difference between the two versions is!

**Comment by ctrlcctrlv** [comment]
https://github.com/ctrlcctrlv/wxsand/blob/5716c16b655ca3670e7acd76372b43763bec20d1/LICENSE#L62-L67

So because of this I knew I didn't have to ask @opiette for permission to redistribute/edit the source code of 4.4.

I'd be happy to distribute 4.5 if you find a version of the source code containing this or a similar license, but it seems like maybe Owen wanted to make 4.5 proprietary? I don't know!

**Comment by BobVul** [comment]
Here's a couple more mods:

[physics.zip](https://github.com/ctrlcctrlv/wxsand/files/3587392/physics.zip)

I think `defaultGradients.png` goes with Ultradensity. Maybe a 4.6 feature. Computer_mod may work on 4.5/4.4. It looks a bit different from the one in your repo, and mentions 4.5 in a comment.

**Comment by BobVul** [comment]
Found a couple more mods in this geocities archive: http://www.oocities.org/normandy_333/FSG/fsg.htm

No luck with the license, unless Owen responds here? :)

**Comment by ctrlcctrlv** [comment]
Thanks! You should also know that my version of wxsand 4.4 has some minor changes not found in the original 4.4. wxWidgets changed a bit in the last ten years.

**Comment by ctrlcctrlv** [comment]
For example, I removed the download functionality that relied on a dead wiki: https://github.com/ctrlcctrlv/wxsand/commit/aa1a0c1662d75622721f978582933fb9a21b85e0

**Comment by Dark-Swarm13** [comment]
i know its been years since anyone added any mods, so i started making my own! i made a sequel to the World Mod by HunterParasite where i doubled the ammount of planets and created a bunch of lore and images for the alien races, that you can find and play here https://drive.google.com/drive/folders/1IalZJtM0V-X5Pln9kljJit8Yy5zie08T?usp=drive_link in this link youll find FSG, the World Mod and World Mod II by me, if anyone has the original mods campaign images that would be awesome! for some reason on windows 10 loading images doesnt work i hope someday that gets fixed! cheers! enjoy the new planets! 
ill be releasing updates there and be making lore videos on YouTube
